lib/dialects/sqlite/query-interface.js

```diff
@@ -32,9 +32,9 @@ module.exports = {
       var sql = self.QueryGenerator.removeColumnQuery(tableName, fields)
         , subQueries = sql.split(';').filter(function(q) { return q !== ''; });
 
-      return Utils.Promise.reduce(subQueries, function(total, subQuery) {
+      return Utils.Promise.resolve(subQueries).each(function(subQuery) {
         return self.sequelize.query(subQuery + ';', null, { raw: true});
```

Comment:
https://github.com/sequelize/sequelize/blob/master/lib/dialects/sqlite/query-generator.js#L324 order is pretty important here too.